### DISCUSSION TITLE
Add ability to label enums in template parameters

### DISCUSSION THREAD

**Discussion by eaglesemanation** [stevearc/overseer.nvim#195] [Ideas]
This is mostly targeted at VSCode input variables of type `"pickString"` with `"options": [{"label": "...", "value": "..."}]`. I have a task that uses absolutely unreadable variables as inputs, so autocompletion doesn't really help in inputs form. I think a good backwards compatible solution would be an ability to specify that enum type will be labeled, and then expect values to be provided as an array of objects (table of tables, Lua is silly).

**Comment by stevearc** [comment]
Yeah, this would be a useful addition. I'll put it on my backlog, and would happily review a PR for it

**Reply by eaglesemanation** [reply]
There is now a PR: #217
